arches/app/utils/data_management/sparql_providers/aat_provider.py

```diff
@@ -26,7 +26,7 @@
 class AAT_Provider(Abstract_Provider):
 
     def __init__(self, **kwargs):
-        super(AAT_Provider, self).__init__('http://vocab.getty.edu/sparql.json', **kwargs)
```

Comment:
This needs to be changed back to use the whole fully qualified url (eg: http://vocab.getty.edu/sparql.json)
This will fail without the protocol specified.